src/lib/crypto/types.ts

```diff
@@ -1,38 +1,60 @@
 const secp256k1 = require('secp256k1')
 import { bech32 } from 'bech32'
-import { Balance, Epoch, Hash, HexString, Nanowits, UtxoMetadata } from "../types"
+import { Balance, Epoch, Hash, HexString, UtxoMetadata } from "../types"
 
-import { fromHexString, fromWits, isHexString, toHexString, whole_wits } from "../../bin/helpers"
+import { fromHexString, isHexString, toHexString, whole_wits } from "../../bin/helpers"
 import { sha256 } from "./utils"
 
 export class Coins {
-    readonly nanowits: Nanowits;
+    protected coins: bigint;
```

Comment:
Should we keep the type `Nanowits` as an alias for `bigint`?

Reply:
as i commented before, having a type for Nanowits is no longer needed, as multiple denominations for $WIT are now encapsulated in the Witnet.Coins class